common/model/catalogue.ts

```diff
@@ -22,6 +22,8 @@ export type Work = {
   collectionPath?: CollectionPath;
   collection?: Collection;
   images?: ImageInclude[];
+  parts?: [];
+  partOf?: [];
```

Comment:
I thought these aren't optional?

Reply:
I think you're right – the code in workDetails expects them to be present.